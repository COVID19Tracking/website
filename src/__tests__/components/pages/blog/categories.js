import React from 'react'
import renderer from 'react-test-renderer'
import Categories from '../../../../components/pages/blog/categories'

describe('Components : Pages : Blog : Categories', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Categories
          categories={[
            {
              name: 'category one',
              slug: 'category-one',
            },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()

    const treeNoCategories = renderer
      .create(<Categories categories={[]} />)
      .toJSON()
    expect(treeNoCategories).toMatchSnapshot()
  })
})
