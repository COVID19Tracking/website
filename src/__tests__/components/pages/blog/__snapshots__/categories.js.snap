// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Categories renders correctly 1`] = `
<div
  className="categoriesContainer"
>
  <p
    className="category"
  >
    <a
      href="/blog/category/category-one"
    >
      category one
    </a>
  </p>
</div>
`;

exports[`Components : Pages : Blog : Categories renders correctly 2`] = `
<div
  className="categoriesContainer"
/>
`;
